packages/ui-react/src/containers/Layout/Layout.tsx

```diff
@@ -1,224 +1,82 @@
+  const navItems = [
+    { label: t('home'), to: '/' },
+    ...menu.map((item) => ({
+      label: item.label,
+      to: playlistURL(item.contentId),
+    })),
+  ];
 
   const containerProps = { inert: sideBarOpen ? '' : undefined }; // inert is not yet officially supported in react
 
   return (
     <div className={styles.layout}>
-      <Helmet>
-        <title>{siteName}</title>
-        <meta name="description" content={metaDescription} />
-        <meta property="og:description" content={metaDescription} />
-        <meta property="og:title" content={siteName} />
-        <meta name="twitter:title" content={siteName} />
-        <meta name="twitter:description" content={metaDescription} />
-      </Helmet>
+      <SiteMetadata />
       <div {...containerProps}>
-        <Header
-          onMenuButtonClick={() => setSideBarOpen(true)}
-          logoSrc={banner}
-          searchEnabled={searchEnabled}
-          searchBarProps={{
-            query: searchQuery,
-            onQueryChange: (event) => updateSearchQuery(event.target.value),
-            onClearButtonClick: () => updateSearchQuery(''),
-            inputRef: searchInputRef,
-          }}
-          searchActive={searchActive}
-          onSearchButtonClick={searchButtonClickHandler}
-          onCloseSearchButtonClick={closeSearchButtonClickHandler}
-          onLoginButtonClick={loginButtonClickHandler}
-          onSignUpButtonClick={signUpButtonClickHandler}
-          onLanguageClick={languageClickHandler}
-          supportedLanguages={supportedLanguages}
-          currentLanguage={currentLanguage}
-          isLoggedIn={isLoggedIn}
-          sideBarOpen={sideBarOpen}
-          userMenuOpen={userMenuOpen}
-          languageMenuOpen={languageMenuOpen}
-          openUserPanel={openUserPanel}
-          closeUserPanel={closeUserPanel}
-          openLanguageMenu={openLanguageMenu}
-          closeLanguageMenu={closeLanguageMenu}
-          canLogin={canLogin}
-          showPaymentsMenuItem={accessModel !== ACCESS_MODEL.AVOD}
-          favoritesEnabled={favoritesEnabled}
-          siteName={siteName}
-          profilesData={{
-            currentProfile: profile,
-            profiles,
-            profilesEnabled,
-            selectProfile: ({ avatarUrl, id }) => selectProfile.mutate({ id, avatarUrl }),
-            isSelectingProfile: selectProfile.isLoading,
-          }}
-          navItems={navItems}
-        />
+        <Header searchActive={searchActive}>
+          <HeaderSkipLink />
+          <HeaderMenu sideBarOpen={sideBarOpen} onClick={openSideBar} />
+          <HeaderBrand siteName={siteName} logoSrc={banner} setLogoLoaded={() => undefined} />
+          <HeaderNavigation navItems={navItems} />
+          <HeaderActions>
+            <HeaderSearch />
+            <HeaderLanguageSwitcher />
+            <HeaderUserMenu />
+          </HeaderActions>
+        </Header>
```

Comment:
Should we probably do it right in the Header.tsx component?

Reply:
That would defeat the purpose. We will have separate copies of the Layout component for each platform. With this, we can change the composition by adding/removing JSX instead of copying all files that need to be changed.